.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
+npm test
```

Comment:
I'm not sure if we should run tests on every commit :thinking: 
Maybe with `pre-push`

Reply:
Is [this](https://www.reactnative.guide/6-conventions-and-code-style/6.2-git-pre-hooks.html) the right way for `pre-push` implementation?
If not will you please refer some.

Reply:
I was expecting just this file with different name, without lint-staged :thinking: